pyproject.toml

```diff
@@ -59,7 +59,6 @@ line-length = 88
 [tool.ruff]
 line-length = 88
 select = ["E", "F", "W", "I", "UP", "N", "B", "A", "C4", "SIM", "ARG"]
-target-version = "py38"
```

Comment:
Shouldn't this remain as `target-version = "py310"`?

Reply:
ruff understands `requires-python` above and `target-version` apparently only duplicates the information